server/routes/api.js

```diff
@@ -3,11 +3,24 @@ const express = require('express')
 const router = express.Router()
 const Rpc = require('../lib/rpc')
 const TransactionCtrl = require('../lib/db/transaction.ctrl')
+const Changelly = require('../lib/changelly/changelly')
 
 router.get('/', (req, res) => res.send('api works'))
 router.get('/rpc/getinfo', Rpc.getInfo)
 router.get('/db/transaction/:id', TransactionCtrl.getTransaction)
 router.get('/db/transaction', TransactionCtrl.getTransaction)
 router.post('/db/transaction', TransactionCtrl.createTransaction)
 
+router.get('/changelly/getCurrencies', (req, res) => Changelly.getCurrencies((err, data) => {
```

Comment:
Can we move this to the changelly script and just have it as a simple function here like the ones above?